Backpack/Label/Classes/BPKLabel.m

```diff
@@ -100,11 +100,22 @@ - (void)setFontMapping:(BPKFontMapping *_Nullable)fontMapping {
     }
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *_Nullable)previousTraitCollection {
+    [super traitCollectionDidChange:previousTraitCollection];
```

Comment:
This code is also gonna be used a lot. Might be worth creating a `Delegate` that components can adhere to to get notified of when they actually need to update appearance.